backend/src/main.rs

```diff
@@ -173,7 +174,11 @@ async fn main() {
             }
         }
         Err(e) => {
-            error!(init_logger, "Failed to open dump"; "error" => format!("{:?}", e));
+            if let ErrorKind::NotFound = e.kind() {
+                // println!("file not found, not logging");
```

Comment:
drop the `println!`?

otherwise, sgtm

Reply:
K. fixed. I could not find a "if not let" to do this. Do u?

Reply:
You can use a match, if you'd like:

```
match e.kind() {
  ErrorKind::NotFound => (),
  _ => {
   // log
  }
}
```

Reply:
Actually, `ErrorKind` implements `Eq`. So you can just do

```
if e.kind() != ErrorKind::NotFound {

}
```

Reply:
Cool. will do. Thanks